modules/gha/scripts/warm-engine.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Make sure not to load any implicit module
+cd $(mktemp -d)
+# Run a simple query to "warm up" the engine
+echo '{directory{id}}' | dagger query
```

Comment:
I found it useful to add `-vvv` to this query so that we see when pre-provisioned Engines are being used, instead of new ones spun up.